scss/utilities/_border.scss

```diff
@@ -0,0 +1,23 @@
+// Border utilities
+.border        { border: 1px $border-gray solid !important; }
+.border-top    { border-top: 1px $border-gray solid !important; }
```

Comment:
why not define this (and those below) as `.border-top { border-top: $border !important; }`?